plugin/evm/syncervm_test.go

```diff
@@ -456,6 +480,15 @@ func testSyncerVM(t *testing.T, vmSetup *syncVMSetup, test syncTest) {
 		}
 	})
 
+	if syncerVM.ethConfig.TxLookupLimit != 0 {
+		tail := syncerVM.blockChain.LastAcceptedBlock().NumberU64() - syncerVM.ethConfig.TxLookupLimit + 1
+
+		syncerVM.blockChain.DrainAcceptorQueue()
+		time.Sleep(50 * time.Millisecond) // Wait for indices initialisation
```

Comment:
is this still needed with the `require.Eventually` added to the check fn?

Reply:
ah good catch, no we should not.